esp/src/src-react/hooks/logging.ts

```diff
@@ -90,7 +91,16 @@ export class ECLWatchLogger implements Writer {
             });
         } else {
             if (_msg instanceof Error) {
-                _msg = _msg.message;
+                let errorMessage = _msg.message;
+                // include cause if present
+                if (_msg.cause) {
+                    if (_msg.cause instanceof Error) {
+                        errorMessage += `\n\t${nlsHPCC.CausedBy}: "${_msg.cause.message}"`;
+                    } else {
+                        errorMessage += `\n\t${nlsHPCC.CausedBy}: "${String(_msg.cause)}"`;
+                    }
+                }
+                _msg = errorMessage;
```

Comment:
I remember discussing adding support for Errors with a "cause" property, but I couldn't find an issue for it... So I just included it this PR.

Printing this ESP exception as is was pretty vague & users wouldn't have any idea why it was happening